src/app/saga.ts

```diff
@@ -205,6 +205,7 @@ export interface RemoteConfigValues {
   dappFavoritesEnabled: boolean
   celoNews: CeloNewsConfig
   twelveWordMnemonicEnabled: boolean
+  dappsDisclaimerMinimal: boolean
```

Comment:
ultra nit: naming is hard, but I wonder if we should use this for clarity
```suggestion
  dappsMinimalDisclaimerEnabled: boolean
```

Reply:
ah, thank you. that is better! i reeeally struggled to name this guy, i'll update